pkg/didcomm/protocol/introduce/service.go

```diff
@@ -22,3 +25,39 @@ const (
 	// AckMsgType defines the introduce ack message type.
 	AckMsgType = IntroduceSpec + "ack"
 )
+
+// Service for introduce protocol
+type Service struct {
+	service.Action
+}
+
+// New returns introduce service
+func New() *Service {
+	return &Service{}
+}
+
+// Handle didexchange msg
+func (s *Service) Handle(msg service.DIDCommMsg) error {
+	return nil
+}
+
+// Name returns service name
+func (s *Service) Name() string {
+	return Introduce
+}
+
+// Accept msg checks the msg type
+func (s *Service) Accept(msgType string) bool {
+	return false
+}
+
+// RegisterMsgEvent on introduce protocol messages. The message events are triggered for incoming messages.
+// Service will not expect any callback on these events unlike Action events.
+func (s *Service) RegisterMsgEvent(ch chan<- service.StateMsg) error {
```

Comment:
Looks like missed to refactor this one. ie, Using [Message](https://github.com/hyperledger/aries-framework-go/pull/449/files#diff-1fb38096269b6a5f914de4f3b8ba6bd8R12)